### PR TITLE
Fix Build

### PR DESCRIPTION
**Pull request by Masquerade0097**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,15 +17,16 @@ build:
 requirements:
   host:
     - pip
-    - python
+    - python >=3.5
     - setuptools
 
   run:
-    - python
+    - python >=3.5
```
Comment (shreyasbapat) [inline recipe/meta.yaml:24]: - python >=3.5
    - numpy >=1.9
    - scipy >=1.0
    - numba >=0.39
    - astropy >=3.1,<4
    - matplotlib >=2.0,!=3.0.1
    - plotly >=3.6,<4

Comment (shreyasbapat) [reply recipe/meta.yaml:24]: Can you try this combination?

File: recipe/run_test.sh [hunk 2]
```diff
@@ -1,3 +1,5 @@
-cd einsteinpy_build
-if errorlevel 1 exit 1
-py.test -vv
+import matplotlib
```
Comment (shreyasbapat) [inline recipe/run_test.sh]: Don't you think you have written Python code in .sh file? I guess it shoudl be renamed

Comment (shreyasbapat) [reply recipe/run_test.sh]: I think just renaming this will pass everything!

Comment (Masquerade0097) [reply recipe/run_test.sh]: 🙈

File: recipe/meta.yaml (hunk 1 above)
Comment (Masquerade0097) [reply recipe/meta.yaml:24]: I have incorporated the changes

**Comment by shreyasbapat** [comment]
@conda-forge-admin, please rerender

**Pull request merged**